nats-streaming/version.txt

```diff
@@ -1 +1 @@
-0.25.6-11
+0.25.6-12
```

Comment:
why versions are incremented for comment update

Reply:
because `apk upgrade --no-cache`